spec/jobs/send_daily_alert_email_job_spec.rb

```diff
@@ -5,7 +5,15 @@
 
   subject(:job) { described_class.perform_later }
 
-  let!(:subscription) { create(:subscription, frequency: :daily) }
```

Comment:
Minor follow up question, can a subscription not have a `search_criteria`?

Reply:
Apparently so.